app/routes/orders/goods.js

```diff
@@ -23,5 +33,12 @@ export default detail.extend({
       data.packages = this.loadPackagesOf(data.order);
       return Ember.RSVP.hash(data);
     });
+  },
+
+  setupController(controller, model) {
+    this._super(controller, model);
+    if (this.get("showMessage")) {
```

Comment:
what if `controller. showUpdateMessage` is already true because of a previous order. Maybe it's worth setting it to false : )

`controller.set("showUpdateMessage", this.get("showMessage"));` would work

(although the comment I wrote above kinda makes this comment pointless :D )